waku/waku_test.go

```diff
@@ -643,13 +644,19 @@ func TestSymmetricSendCycle(t *testing.T) {
 	InitSingleTest()
 
 	w := New(nil, nil)
-	defer w.SetMinimumPoW(common.DefaultMinimumPoW, false)  // nolint: errcheck
-	defer w.SetMaxMessageSize(common.DefaultMaxMessageSize) // nolint: errcheck
+	defer func() {
+		handleError(t, w.SetMinimumPoW(common.DefaultMinimumPoW, false))
+	}()
+	defer func() {
+		handleError(t, w.SetMaxMessageSize(common.DefaultMaxMessageSize))
+	}()
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)